.github/workflows/build-test.yml

```diff
@@ -216,10 +216,15 @@ jobs:
         with:
           go-version: '1.17'
 
+      - name: Generate Gremlin Server Base Image
+        working-directory: .
+        run: |
+          mvn clean install -pl :gremlin-server -DskipTests -DskipIntegrationTests=true -am && mvn install -Pdocker-images -pl :gremlin-server
+
       - name: Execute Go tests
         working-directory: ./gremlin-go
         run: |
-          docker-compose up --exit-code-from gremlin-go-integration-tests gremlin-go-integration-tests
+          docker-compose up --exit-code-from gremlin-go-integration-tests
```

Comment:
why does this need to change?

Reply:
Aren't they the same service? They were placed twice before.

Reply:
yep this was supposed to be changed a while back, we never got around to doing it